project/src/main/java/com/google/sps/servlets/dataAnalytics.java

```diff
@@ -78,13 +77,13 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
       Entity commentEntity = new Entity("daily_temp", day);
       commentEntity.setProperty("temp", total/num_temps);
-      commentEntity.setProperty("school", person.substring(person.indexOf("school\":")+9, person.indexOf("\",\"phone")));
+      commentEntity.setProperty("school", school);
       datastore.put(commentEntity);
       String val = Double.toString(total/num_temps);
       ArrayList<String> data = new ArrayList<String>();
       data.add(val);
       data.add(Integer.toString(fevers));
-      if (person.substring(person.indexOf("admin\":")+7, person.indexOf("}")).equals("true")) {
+      if ((boolean) session.getAttribute("admin")) {
```

Comment:
I would like the attribute named "is_admin". Boolean value for "admin" is not very obvious. If it's not very time consuming - consider to change it to is_admin.

Reply:
I can implement that, I'll just have to let Thomas and Christina know to change their session.getAttribute() usage to "is_admin" instead of "admin".